src/seedu/addressbook/data/person/TimeTable.java

```diff
@@ -0,0 +1,49 @@
+package seedu.addressbook.data.person;
+
+public class TimeTable {
+
+    private TimeSlot[][] timeslot;
+    private boolean isPrivate;
+    private boolean isPublic;
+    private boolean isFinal;
+    private boolean isNotFinal;
+
+    public TimeTable(){
+
+    }
+
```

Comment:
Missing header comment. All non-trivial methods should have java doc format header comments.